WcaOnRails/app/models/competition.rb

```diff
@@ -27,7 +27,7 @@ class Competition < ApplicationRecord
            allow_nil: true,
            with_model_currency: :currency_code
 
-  scope :not_over, -> { where("end_date >= ?", Date.today) }
+  scope :not_over, -> { where("end_date >= ? AND results_posted_at IS NULL", Date.today) }
```

Comment:
You could probably rewrite `is_probably_over?` to use the `not_over` scope. The price is an extra DB query.

Are they really synced when `end_date.nil?`?